packages/host/tests/integration/components/serialization-test.gts

```diff
@@ -5588,14 +5600,14 @@ module('Integration | serialization', function (hooks) {
           attributes: { firstName: 'Burcu' },
           relationships: {
             friend: { links: { self: `${testRealmURL}Friend/hassan` } },
-            'friendPets.0': {
+            'ownPets.0': {
               links: { self: `${testRealmURL}Pet/mango` },
               data: {
                 id: `${testRealmURL}Pet/mango`,
                 type: 'card',
               },
             },
-            'friendPets.1': {
+            'ownPets.1': {
               links: { self: `${testRealmURL}Pet/vanGogh` },
               data: {
                 id: `${testRealmURL}Pet/vanGogh`,
```

Comment:
ditto with stomping on top of computeds and asserting that bug in CS-8486 exists!